src/activities/guesscount-summer/dataset.js

```diff
@@ -0,0 +1,59 @@
+/* GCompris - dataset.js
+ *
+ * Copyright (C) 2016 Rahul Yadav <rahulyadav170923@gmail.com>
+ *
+ * Authors:
+ *   <Pascal Georges> (GTK+ version)
+ *   RAHUL YADAV <rahulyadav170923@gmail.com> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+// questions according to operators
+
+var dataset=[
+            [
+                [['+'],[  [[1,2],3] , [[3,4],7] ,[[6,2],8] ,[[5,2],7] ,[[9,2],11] ,[[4,2],6],[[1,6],7] ,[[3,2],5] ,[[13,1],14] ,[[5,8],13]  ]],
+                [['-'],[  [[2,1],1] , [[3,1],2] ,[[7,2],5] ,[[6,2],4] ,[[9,2],7] ,[[8,5],3],[[9,5],4],[[18,9],3],[[5,1],4],[[6,3],3],[[5,4],1] ]],
```

Comment:
[[18,9],9]
for all the dataset, you should add fake numbers (at least one for first level) else it's too easy with only two numbers to guess the result.